dali/python/bundle-wheel.sh

```diff
@@ -124,6 +124,8 @@ DEPS_LIST=(
     "${DEPS_PATH}/lib/libcfitsio.so.4"
     "lib/libcvcuda.so.0"
     "lib/libnvcv_types.so.0"
+    "lib/libcvcuda_d.so.0"   # debug build
+    "lib/libnvcv_types_d.so.0"   # debug build
```

Comment:
May I ask for making this comment more descriptive?

Reply:
```bash
    # cvcuda adds _d suffix to lib names for debug builds
    "lib/libcvcuda_d.so.0"
    "lib/libnvcv_types_d.so.0"
```
descriptive enough?